app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,83 @@
+<div class="container hp">
+
```

Comment:
### Optional
- [ ] It would be better if we add 2 buttons or links, that is 1. Generate shopping list and 2. Add Ingredients as we have in the wireframe. These are missing from our UI, kindly compare the images below
![Screenshot from 2023-08-13 01-40-51](https://github.com/zamcham/TasteDiary/assets/46388113/66fbfe75-87ed-4da7-83f0-31968a6a70c9)
![Screenshot from 2023-08-13 01-40-19](https://github.com/zamcham/TasteDiary/assets/46388113/8464aed8-7604-4284-9f23-64f0aeae333d)

Reply:
Hello @addod19, Thank you very much for the quick review. The buttons are present when a user is signed in because a shopping list can only be generated by a signed in user and ingredients can only be added if the user is the owner of the recipe. 
Thank you very much again and have an awesome weekend.